binary/bitstring.rkt

```diff
@@ -20,6 +20,7 @@
   [bitstring->padded-bytes (-> bitstring? immutable-bytes?)]
   [bytes->bitstring
    (->* (immutable-bytes?) (#:padding (integer-in 0 7)) bitstring?)]
+  [sequence->bitstring (-> (sequence/c bit?) bitstring?)]
```

Comment:
To avoid reconverting bitstrings, we also have to avoid wrapping them in a `sequence/c` contract.
```suggestion
  [sequence->bitstring (-> (or/c bitstring? (sequence/c bit?)) bitstring?)]
```